### PR TITLE
HUBManager: Add convenience initializer

### PR DESCRIPTION
**Pull request by JohnSundell**
Since we want to support a lot of customization options for the Hub Framework (we also need them for the Spotify app), the initializer for `HUBManager` is quite large. However, most of its arguments are optional, so with this change a convenience initializer is added that only takes the required arguments.

Also update the demo app to using this new initializer.

### CODE REVIEW

**Comment by JohnSundell** [comment]
@spotify/objc-dev

**Comment by JohnSundell** [comment]
@Goos Me too 😢

File: sources/HUBManager.m [hunk 1]
```diff
@@ -47,6 +47,19 @@ @implementation HUBManager
 
 - (instancetype)initWithComponentLayoutManager:(id<HUBComponentLayoutManager>)componentLayoutManager
                       componentFallbackHandler:(id<HUBComponentFallbackHandler>)componentFallbackHandler
+{
+    return [self initWithComponentLayoutManager:componentLayoutManager
+                       componentFallbackHandler:componentFallbackHandler
+                      connectivityStateResolver:nil imageLoaderFactory:nil
```
Comment (rastersize) [inline sources/HUBManager.m]: Move `imageLoaderFactory:nil` to its own line.

Comment (JohnSundell) [inline sources/HUBManager.m]: Ah of course, silly mistake! Will fix 👍

**Pull request merged**